src/nxt_http_request.c

```diff
@@ -549,6 +549,34 @@ nxt_http_request_ready(nxt_task_t *task, void *obj, void *data)
     r = obj;
     action = r->conf->socket_conf->action;
 
+    if (r->chunked) {
+        r->transfer_encoding->skip = 1;
+        size_t size = r->body->file_end;
+
+        nxt_http_field_t *f;
+        f = nxt_list_zero_add(r->fields);
+        if (nxt_slow_path(f == NULL)) {
+            /* error */
+            return;
+        }
+
+        nxt_http_field_name_set(f, "Content-Length");
+
+        u_char *p, *end;
```

Comment:
Can we please declare these at the top of the `if ()` scope?, i.e

```c
if (r->chunked) {
    u_char  *p, *end;

...
```

(I'd love for us to be able to compile with `-Wdeclaration-after-statement`)

Reply:
Yep, such declares should be at the top.
I guess the code will be rewrite with a good design, now it's for the running demo.